lib/pbench/common/exceptions.py

```diff
@@ -52,3 +52,9 @@ class BadSampleName(Exception):
     """
 
     pass
+
+
+class FileUploadError(Exception):
```

Comment:
Is this really a common exception?  Seems like it is only raised by `CopyResultTb`.  Why not have it as an exception of that class?

Reply:
yeah, I will just move it :)